### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## Below is the description of the pull request

- [ ] Add the following validations for the user model
- Name must not be blank.
- PostsCounter must be an integer greater than or equal to zero.
- 
- [ ] Add the following validations for the post model
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

- [ ] Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -11,4 +11,7 @@ def update_post_counter
   def recent_comments
     comments.order(created_at: :desc).limit(5)
   end
+  validates :Title, presence: true, length: { in: 6..250 }
+  validates :comments_counter, presence: true, numericality: { only_integer: true }
+  validates :likes_counter, presence: true, numericality: { only_integer: true }
```
Comment (Whoistolu) [inline app/models/post.rb:16]: - [ ] [REQUIRED] Great job @Kaweesi-Matia , However, a part of the project requirement states that the `CommentsCounter` must be an integer greater than or equal to zero, and also the `LikesCounter` must be an integer greater than or equal to zero. Kindly implement this as you have implemented it in the User model validation for the `PostCounters`.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Kaweesi-Matia 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/post.rb (hunk 1 above)
Comment (Kaweesi-Matia) [reply app/models/post.rb:16]: Implement validations as required by the reviewer

**Review by AshakaE — APPROVED** [review]
Hi @Kaweesi-Matia , 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**